### PR TITLE
Fix Windows import library name

### PR DESCRIPTION
**Pull request by jamrial**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -80,7 +80,7 @@ ifdef CONFIG_SHARED
     LDFLAGS_DLL = $(LDFLAGS) -shared
 
     ifdef CONFIG_WINDOWS
-        IMP_LIB = libdcadec/libdcadecdll$(LIBSUF)
+        IMP_LIB = libdcadec/libdcadec.dll$(LIBSUF)
```
Comment (TimothyGu) [inline Makefile]: How about `libdcadec$(DLLSUF)$(LIBSUF)`?

Comment (jamrial) [inline Makefile]: It's cosmetics. I don't want to redo the pull request unless there's a problem, and it can always be changed later.

Comment (TimothyGu) [inline Makefile]: No problem. I was just suggesting since you are changing this line anyway.

**Comment by foo86** [comment]
Merged, thanks.

**Pull request closed without merging**